py/torch_tensorrt/fx/passes/pass_utils.py

```diff
@@ -179,6 +179,27 @@ def bounded_method(*args, **kwargs):
     return dec_for_method
 
 
+def req_torch_version(min_torch_version: str = "2.0") -> Decorator:
+    def nested_decorator(f: Callable):
+        def function_wrapper(*args, **kwargs):
+            # Parse minimum and current Torch versions as tuples of strings
+            # Split out GPU version (denoted via +_____)
+            reqd_torch_version = tuple(min_torch_version.split("+")[0].split("."))
+            current_torch_version = tuple(torch.__version__.split("+")[0].split("."))
+
+            if current_torch_version < reqd_torch_version:
```

Comment:
Lets print the name of the function so users are informed that its just this part of the library that is not supported.

Reply:
In the raised error, the function itself is printed directly. This includes the name of the function as well.